core/src/main/java/com/google/zxing/datamatrix/decoder/DecodedBitStreamParser.java

```diff
@@ -117,15 +126,34 @@ static DecoderResult decode(byte[] bytes) throws FormatException {
     if (resultTrailer.length() > 0) {
       result.append(resultTrailer);
     }
-    return new DecoderResult(bytes, result.toString(), byteSegments.isEmpty() ? null : byteSegments, null);
+    if (isECIencoded) {
+      if (fnc1Positions.containsKey(0) || fnc1Positions.containsKey(4)) {
+        symbologyModifier = 5;
```

Comment:
If you have a link explaining these numbers, that would be good to include in a comment

Reply:
There are links, but they all have caveats:
- The standard ISO/IEC15424 is not publicly available (ISO paywall).
- The GS1 Specification https://www.gs1.org/docs/barcodes/GS1_General_Specifications.pdf has a list in section 5.1.3, but only lists the GS1 specific ones.
- https://honeywellaidc.force.com/supportppr/s/article/List-of-barcode-symbology-AIM-Identifiers has a more complete list, but is actually part of the documentation of a specific hardware barcode scanner product.